ddd/preparation/projet_doctoral/domain/service/profil_candidat.py

```diff
@@ -23,10 +23,11 @@
 #    see http://www.gnu.org/licenses/.
 #
 # ##############################################################################
+from admission.ddd.preparation.projet_doctoral.domain.model._candidat_adresse import CandidatAdresse
 from admission.ddd.preparation.projet_doctoral.domain.model._candidat_signaletique import CandidatSignaletique
 from admission.ddd.preparation.projet_doctoral.domain.service.i_profil_candidat import IProfilCandidatTranslator
 from admission.ddd.preparation.projet_doctoral.domain.validator.validator_by_business_action import (
-    IdentificationValidatorList,
+    IdentificationValidatorList, CoordonneesValidatorList,
```

Comment:
En multiligne

Pycharm > Settings > Code style python > Wrapping and braces
![Screenshot 2022-02-11 at 14 06 19](https://user-images.githubusercontent.com/645207/153596622-533b7aa8-ea70-409b-ae8c-db460ead6d26.png)